src/App.jsx

```diff
@@ -40,18 +41,44 @@ function App() {
         }
       }
     }, {})
-  }, [responseArray])
+  }, [reviveResponseArray])
 
-  const reviveData = useMemo(() => {
-    return Object.values(usersRevivedAsObject)
-  }, [usersRevivedAsObject])
+  const parsedUserEventsAsObject = useMemo(() => {
+    const getIdRegex = /XID\=([0-9]+)(?:"\>|\>)(.+)\<.*/gm;
+    const stripAnchorRegex = /(<\/?[a|A][^>]*>|)/gm;
```

Comment:
NIT: These could be defined as constants (usually in `SCREAMING_SNAKE_CASE`) at the top of the file.

The primary reason for this is that it stops the variables being re-instantiated between renders.